packages/bullet-list/example/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html style="height: 100%">
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Bullet list</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+    </head>
+
+    <body style="height:100%;display:flex;align-items:center;justify-content:center;">
```

Comment:
Opionion: I think it is better to put all styles and style imports in a `index.scss` like in button-example.

Reply:
Now it is some inline in html and some style imports in the react component.